server/src/main/resources/db/common/V91__alter_feedback_answer_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE feedback_answers
+ADD CONSTRAINT feedback_answers_uniqueconstraint UNIQUE (question_id, request_id);
```

Comment:
This is going to need to change to V92 to accommodate the migration in #1257

Reply:
Fixed